src/app/components/forms/input-text-readonly.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input } from "@angular/core";
+
+import { PlatformUtilsService } from "jslib-common/abstractions/platformUtils.service";
+
+@Component({
+  selector: "app-input-text-readonly",
+  templateUrl: "input-text-readonly.component.html",
+})
+export class InputTextReadOnlyComponent {
+  @Input() controlValue: string;
+  @Input() label: string;
+  @Input() showCopy: boolean = true;
+  @Input() showLaunch: boolean = false;
```

Comment:
This component doesn't feel quite reuseable enough for it to be the definitive input-text-readonly. I suspect we will need to do something similar to https://material.angular.io/components/form-field/overview

Reply:
I'll have a go at refactoring these components to use that pattern. I think it's a good dry-run for the component library, then we can migrate them later.